assets/src/scss/bootstrap-build.scss

```diff
@@ -20,6 +20,7 @@ Text Domain: planet4-master-theme
 
 @import "~bootstrap/scss/functions";
 @import "~bootstrap/scss/variables";
+@import "./bootstrap-variables";
 @import "~bootstrap/scss/mixins";
```

Comment:
Moving that as the first import seems to work even for completely removing `xxl` breakpoint.